openshift/release/create-release-branch.sh

```diff
@@ -24,6 +24,8 @@ PATCH_DIR="openshift/patches"
 # Use release-specific patch dir if exists
 if [ -d "openshift/patches-${release}" ]; then
     PATCH_DIR="openshift/patches-${release}"
+    # Update the nightly test images to actual versioned images
+    sed -i "s/knative-nightly:knative/knative-${release}:knative/g" openshift/patches/*.patch
```

Comment:
We use the $PATCH_DIR variable now 

```suggestion
    sed -i "s/knative-nightly:knative/knative-${release}:knative/g" ${PATCH_DIR}/*.patch
```

Reply:
Nice catch! LOL! I used my local `sed` fu to test this ... thanks for the :eye:  on this!